Resizing banner images to matching heights

Ldsmiths opened the pull request:
refer issue #273 

-New banner images for all Stratmaps tabs to make same height
-Youtube video aspect ratio changed to more closely match banner height (Stratmap Land Parcels & Address Points tabs)
-Slight modifications to Education & Training layout to match Teacher Resources page

geochik commented:
Thank you, James!

Sent from my iPhone

> On Mar 31, 2015, at 3:54 PM, James Seppi <notifications@github.com> wrote:
> 
> Merged #274.
> 
> —
> Reply to this email directly or view it on GitHub.
>

jseppi commented:
No problem.
Merged and deployed.

Commit:
issue #273

-New banner images for all stratmaps tabs to make same height
-Youtube video aspect ratio changed to more closely match banner height (Stratmap Land Parcels & Address Points tabs)
-Slight modifications to Education & Training layout to match Teacher Resources page

jwhaney reviewed (approved):
looks good 👍
